editor/import/resource_importer_texture.cpp

```diff
@@ -276,11 +285,9 @@ void ResourceImporterTexture::save_to_ctex_format(Ref<FileAccess> f, const Ref<I
 				} else {
 					data = Image::png_packer(i ? p_image->get_image_from_mipmap(i) : p_image);
 				}
-				int data_len = data.size();
-				f->store_32(data_len);
 
-				const uint8_t *r = data.ptr();
-				f->store_buffer(r, data_len);
+				f->store_32(data.size());
+				f->store_buffer(data);
```

Comment:
This is removing a minor optimization, forcing `store_buffer` to recalculate `data`'s size, when we just did previously:
```cpp
bool FileAccess::store_buffer(const Vector<uint8_t> &p_buffer) {
        uint64_t len = p_buffer.size();
        if (len == 0) {
                return true;
        }

        const uint8_t *r = p_buffer.ptr();

        return store_buffer(r, len);
}
```

Is the minor readability gain worth the (minor?) loss in performance?